deployment.md

```diff
@@ -0,0 +1,41 @@
+# Azure Resources
+
+Create and configure the following resources
+
+## Python
+
+- Storage account
+  - Security + networking -> Access keys, copy key1 value to use in the Function App
+- Cosmos DB
+  - Settings -> Keys blade, copy the Primary Connection String to use in the Function App
+- Function App
+  - Runtime stack: Python v3.9
+  - Version: ~3
+  - Settings -> Configuration blade, create the following application settings:
+    - COSMOS_CONNECTION: Key copied from CosmosDB
+    - CONTEXT_STORAGE_ACCOUNT_CONNECTION: Key copied from Storage Account
```

Comment:
Let's use : 

CONTEXT_STORAGE_ACCOUNT_CONNECTION: Storage Account access key (available from Security & Networking Tab)

and remove the line that says `- Security + networking -> Access keys, copy key1 value to use in the Function App`